code/modules/urist/decals/urist.dm

```diff
@@ -45,3 +45,68 @@
 /obj/structure/sign/directions/urist/toolstorage
 	name = "\improper Tool Storage Directions"
 	icon_state = "direction_toolstorage"
+
+// Paintings:
+
+/obj/structure/sign/painting
+	icon = 'icons/urist/decals/urist.dmi'
+
+/obj/structure/sign/painting/dogpoker
+ 	name = "\improper dog painting"
+ 	desc = "A painting of some dogs playing poker, wonder if Ian is in it."
+ 	icon_state = "dogpoker"
+
+/obj/structure/sign/painting/starry
+ 	name = "\improper beautiful landscape painting"
+ 	desc = "A beautiful painting with swirling stars and a twisting skyline."
+ 	icon_state = "starry"
+
+/obj/structure/sign/painting/snow
+ 	name = "\improper snowy painting."
+ 	desc = "A calming picture of a cabin surronded by velvety snow."
+ 	icon_state = "snowy"
+
+/obj/structure/sign/painting/gogh
+ 	name = "\improper portrait"
+ 	desc = "An impressive self-portrait of an artist, dated many centuries ago."
+ 	icon_state = "gogh"
+
+/obj/structure/sign/painting/bigwave
+ 	name = "\improper tidal wave painting"
+ 	desc = "A painting of a huge wave, written in an old ancient language you cannot seem to understand."
+ 	icon_state = "wave"
+
+/obj/structure/sign/painting/desert
+ 	name = "\improper desert painting"
+ 	desc = "A picture of what appears to be bones cast out in the desert, it looks quite melancholy and haunting.."
+ 	icon_state = "desert"
+
+/obj/structure/sign/painting/stillife
+	name = "\improper still-life painting"
+	desc = "A painting that shows what appears to be fruit on a table in great detail."
+	icon_state = "still"
+
+/obj/structure/sign/painting/persistance
+	name = "\improper surreal painting"
+	desc = "A surreal painting focusing on clocks melting into the landscape, almost trippy to look at."
+	icon_state = "persistance"
```

Comment:
"Persistence", rather. It's bugging me a bit, but you'd have to fix up the DMI as well.

Reply:
fixed